### PR TITLE
commands: various semantic cleanups and corrections, no functional ch…

### PR DESCRIPTION
**Pull request by rpjday**
…ange

In some cases, tighten up the help messages and docstrings as there is little value in verbosity in the code.

- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `make lint`?
- [ ] Have you successfully run `pytest tests/unit`?

-----

### CODE REVIEW

**Review by tigarmo — APPROVED** [review]
Very helpful, thanks!

**Review by mr-cal — APPROVED** [review]
Thanks!

File: snapcraft/commands/validation_sets.py [hunk 1]
```diff
@@ -60,10 +60,10 @@
 
 
 class StoreEditValidationSetsCommand(BaseCommand):
-    """Command for edit-validation-sets."""
+    """Edit a validation set.."""
```
Comment (mr-cal) [inline snapcraft/commands/validation_sets.py:63]: I wish I noticed this prior to all the tests passing!

Can you remove the extra period here?

**Comment by rpjday** [comment]
As an amended commit?

**Comment by mr-cal** [comment]
Yep, that works, thanks!

**Pull request merged**